src/main.ts

```diff
@@ -75,5 +77,6 @@ fetchConfig("config.json").then((config: any) => {
   app.provide("$store", store)
   app.config.globalProperties.$lokapi = lokApiService
   app.config.globalProperties.$config = config
+  app.config.globalProperties.$pStore = new LocalStore("monujo")
```

Comment:
Just for my information, can you explain to me what `pStore` means to you ? Persistent store maybe ?

Reply:
yes the p is for persistent, i am going to give it more explicit name

Reply:
If you prefer to change it, up to you. I'm fine with this name, as long as we can understand the underlying meaning.